.github/workflows/production-tag.yml

```diff
@@ -0,0 +1,128 @@
+name: Create Production Tag
+
+on:
```

Comment:
This intention here is that 2 things will trigger a new production tag:
* manual tagging (this is a backup)
* a completed successful run of the CI workflow on the main branch

This is to get away from daily timed tagging and enable more continual deployment with tests in place to ensure application integrity.